docs/pages/guides/schema-validation.mdx

```diff
@@ -64,13 +80,140 @@ of the data you want to store in your rooms.
   </video>
 </Figure>
 
+Another way to create a schema is via the
+[create schema API](/docs/api-reference/rest-api-endpoints#post-create-new-schema).
+When a schema is sent to the API it needs to be constructed as a string literal
+to be valid JSON when sent to the API for parsing. In the dashboard, a sample
+schema could be represented as:
```

Comment:
The schema body needs no special treatment and we shouldn't put too much emphasis on it. The trouble you ran into came mostly from the fact that you tried to use cURL, and newline characters give problems in a command line environment and in raw JSON, in case you would write that.

Typically, people will send these requests from JavaScript though, in which case `JSON.stringify()` will handle strings with newlines just like any other string, and it won't be an issue.

That said, when you are explaining how to make a cURL request, then it may be useful to mention this. In this case, it's not needed.

Suggestion:

```suggestion
A schema body is a multi-line string using the Liveblocks <link>schema syntax</link>. In the dashboard, a sample
schema could be represented as:
```